Dockerfile

```
@@ -1,13 +1,36 @@
-FROM node:10.15-slim
+FROM node:alpine
```

Could we please use a fixed version node version instead of latest?

```suggestion
FROM node:14.15.4-alpine
```